test/dataframerow.jl

```diff
@@ -71,8 +90,22 @@ end
     @test copy(view(r, [:c,:b])) == (c = "CC", b = 1.2)
     @test copy(view(r, [:c,:b])) == (c = "CC", b = 1.2)
     @test copy(r[[false, true]]) == (c = "CC",)
+    @test copy(r[r"b"]) == (b = 1.2,)
```

Comment:
Why not test `r"[cb]"` for consistency with the test above?

Reply:
added.